docs/Get-HelpPreview.md

```diff
@@ -1,30 +1,29 @@
 ---
 external help file: platyPS-help.xml
 schema: 2.0.0
-online version: https://github.com/PowerShell/platyPS/blob/master/docs/Get-HelpPreview.md
```

Comment:
This happened because you made changes to the cmdlet between the pull request and the present. Putting the online version in the metadata. I'll fix this and the other stuff you found and update the commit. I should be able to get to it today - Thanks -